src/org/omegat/Main.java

```diff
@@ -45,11 +45,11 @@
 import java.util.Map;
 import java.util.PropertyResourceBundle;
 import java.util.TreeMap;
-
 import javax.swing.JOptionPane;
 import javax.swing.SwingUtilities;
 import javax.swing.UIManager;
 
+import com.vlsolutions.swing.docking.DockingDesktop;
```

Comment:
Your IDE seems to prefer different import-ordering than we tend to use in OmegaT.

I don't really like this unnecessary churn.

Can you tell me what IDE you're using? I believe we have presets for Eclipse; maybe we can figure out something for your IDE as well.

Reply:
I use IntelliJ IDEA.

Reply:
Fixed import order.